Source/Core/Core/NetPlayClient.cpp

```diff
@@ -514,7 +533,7 @@ void NetPlayClient::ThreadFunc()
 				break;
 			}
 		}
-
+		Common::SleepCurrentThread(1);
```

Comment:
Better to leave it for now i think because this cripples performance without it. These PRs will conflict either way anyway. I think it is best to avoid such a regression until that pr is merged, assuming this is merged first.